### PR TITLE
Refactored users helper and added unit tests

### PR DESCRIPTION
**Pull request by KevinMulhern**
This is a refactor of the users helper, it also adds unit tests for this helper.

Fixes:
https://github.com/TheOdinProject/theodinproject/issues/222

### CODE REVIEW

**Comment by KevinMulhern** [comment]
@arun1595 Review please 😄

File: app/helpers/users_helper.rb [hunk 1]
```diff
@@ -1,48 +1,32 @@
...
-    url = ""
-    if user.twitter
-      if user.twitter[0] == "@"
-        url = "http://www.twitter.com/#{user.twitter[1..-1]}"
-      else
-        url = "http://www.twitter.com/#{user.twitter}"
-      end
-    end
-  end 
+    social_media_link.new(user).twitter_url
```
Comment (arun1595) [inline app/helpers/users_helper.rb]: Can we not use `SocialMediaLink` directly? Do you have any reason to do it this way? 

IMO, 
```ruby
SocialMediaLink.new(user).twitter_url
```
looks more clear.

**Comment by KevinMulhern** [comment]
**NOTE** tests pass locally, CI is still acting up :(

File: app/helpers/users_helper.rb (hunk 1 above)
Comment (KevinMulhern) [inline app/helpers/users_helper.rb]: I did that way incase the class name changes in the future (which its unlikely to do) I think it reads cleaner with the classes constant in the method too. I'll update it.

**Pull request merged**